src/components/index/OpenSource.tsx

```diff
@@ -1,100 +1,44 @@
-            }
-
-            &-container {
-                display: flex;
-                align-items: center;
-                flex: 0 0 55%;
-            }
-        }
-
-        &__text {
-            
-            &-container {
-                display: flex;
-                align-items: center;
-                flex: 1;
-                max-width: 60rem;
-
-                @media(min-width: 1040px) {
-                    padding: 0 0 0 6rem; 
-                }
-            }
-
-            p {
-                margin-bottom: 2rem;
-            }
-
-            @media(max-width: ${sizes.breakpoints.lg}) {
-                br {
-                    display: none;
-                }
-            }
-        }
+    h2 + p {
+        margin: 0;
+    }
 
-        h2 {
-            color: ${colors.textDark};
-            margin-bottom: 6rem;
-            font-weight: 600;
+    .text {
+        max-width: 600px;
+        margin: auto;
+        padding: 10rem 6rem 6rem;
+        background: ${colors.white};
+        border: ${borders.light};
+        border-radius: 3px;
+    }
 
-            @media(min-width: ${sizes.breakpoints.md}) {
-                font-size: 200%;
-            }
-        }
+    img {
+        display: inline-block;
+        height: 8rem;
+        margin-bottom: 5rem;
     }
 `
 
 const OpenSource = () => (
     <Styled>
-        <div className="grey-container">
-            <section>
-                <div className="row">
-                    <div className="open-source">
-                        <div className="open-source__img-container">
-                            <img className="open-source__img" src={OpenSourceImg} />
-                        </div>
-                        <div className="open-source__text-container">
-                            <div className="open-source__text">
-                                <h3>Giving Back to Open-Source!</h3>
-                                <p>Gitpod is built on open-source and wouldn’t exist without it.</p> 
-                                <p>Therefore, Gitpod is free for open source, and we have created contribute.dev, which lists ready-to-code open source projects. Start contributing now!</p> 
-                                <p>Find your next project!</p>
-                            <a href="https://contribute.dev" className="btn btn--cta" target="_blank" rel="noopener">Contribute.dev</a>
-                            </div>
-                        </div>
-                    </div>
-                </div>
-            </section>
+        <div className="row">
+            <div className="text">
+                <img src={OpenSourceImg} alt=""/>
+                <h2><strong>Giving Back to Open-Source!</strong></h2>
+                <p>Gitpod is built on open-source and wouldn’t exist without it. We’re happy to give something back by supporting the open-source community with a <Link to="/docs/professional-open-source/">free Open-Source Plan</Link>.</p>
```

Comment:
We're not really giving out free plans to everyone. We're sponsoring professional open-source developers with unlimited hours.

```suggestion
                <p>Gitpod is built on open-source and wouldn’t exist without it. We’re happy to give something back by sponsoring <Link to="/docs/professional-open-source/">Professional Open-Source</Link> developers with unlimited Gitpod hours on open-source repositories.</p>
```

Reply:
This is referring to the free 50hours open-source plan and not to the professional open-source plan. The professional plan only affects a few users while the free open-source plan is relevant for everyone. That's why I thought it makes more sense to mention it. 

Maybe instead we could mention the 50hours/ month limitation.

Reply:
> This is referring to the free 50hours open-source plan and not to the professional open-source plan.

Ah, in this case maybe:

```suggestion
                <p>Gitpod is built on open-source and wouldn’t exist without it. We’re happy to give something back by being completely <strong>free for open source</strong>.</p>
```